docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -1,11 +1,45 @@
 # Converting a PyTorch Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch}
 
-The PyTorch framework is supported through export to the ONNX format. In order to optimize and deploy a model that was trained with it:
+The PyTorch framework is supported through export to the ONNX format. Model Optimizer Python API allows the conversion of PyTorch models using the `convert_model()` method, which internally converts a model to ONNX.
+
+## Converting a PyTorch model from memory with Python API (Experimental Functionality)
+
+To convert a PyTorch model using `convert_model()`, provide the `input_shape` or `example_input`.
+
+```sh
+import torchvision
+import torch
+from openvino.tools.mo import convert_model
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, example_input=torch.zeros(1, 3, 100, 100))
+```
+
+'example_input' accepts the following formats:
+
+* `openvino.runtime.Tensor`
+* `torch.Tensor`
+* `np.ndarray`
+* `list` or `tuple` with tensors (`openvino.runtime.Tensor` / `torch.Tensor` / `np.ndarray`)
+* `dictionary` where key is the input name, value is the tensor (`openvino.runtime.Tensor` / `torch.Tensor` / `np.ndarray`)
+
+ONNX opset version can be set using an optional `onnx_opset_version` parameter.
+If the `onnx_opset_version` is not set, the default opset from `torch.onnx.export()` is used.
+
+```sh
+import torchvision
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, input_shape=[1,3,100,100], onnx_opset_version=13)
+```
+
+## Exporting a PyTorch Model to ONNX Format <a name="export-to-onnx"></a>
+
+For complicated cases, PyTorch models can be first exported to ONNX before MO conversion:
```

Comment:
What do you mean by `complicated cases`? Please be more specific, for example when `convert_model` fails to convert your model you can convert to ONNX first with specific torch.onnx.export parameters which are not available in convert_model, explore the model and apply model cutting techniques (with input/output parameters) if there operations that are not supported by OpenVINO.

Reply:
Replaced this line with your suggestion.